### ISSUE TITLE
Read instance for strings w/o timezone has unclear fatal error message

### ISSUE THREAD

**Issue by jmtd** [haskell/time#207]
With [time-1.8.0.2](https://www.stackage.org/lts-13.25/package/time-1.8.0.2):
```
λ> read "2013-01-01 00:00:00" :: UTCTime
2013-01-01 00:00:00 UTC
```

With [time 1.9](https://www.stackage.org/lts-19.0/package/time-1.9.3):
```
λ> read "2013-01-01 00:00:00" :: UTCTime
*** Exception: Prelude.read: no parse
```

There was nothing obvious in the changelog, except

 * parsing: reject invalid (and empty) time-zones with %z and %Z 

adding a TZ specifier works:

```
λ> read "2013-01-01 00:00:00 +0000" :: UTCTime
2013-01-01 00:00:00 UTC
```

But since I'm not  - directly - using the parsing function (indirectly via Read), this wasn't immediately obvious. The runtime error "no parse" is also not helpful. Is there any chance this can be improved?

**Comment by AshleyYakeley** [comment]
This is already fixed in 1.11.1.1. (Not sure which version fixed it.)
```
ghci> read "2013-01-01 00:00:00" :: UTCTime
2013-01-01 00:00:00 UTC
```

**Comment by jmtd** [comment]
Thank you: Sorry, it didn't occur to me to check the latest version (It should have). I had been thinking, in my case, these are basically static instances of UTCTime used for testing purposes. More ideal than relying on `read` would perhaps be hard-coding the UTCTime instance using constructors, but that proved complex enough to unwind that I gave up. I should look at that again.
